extensions-core/kubernetes-extensions/pom.xml

```diff
@@ -137,6 +138,19 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
+        <plugins>
+          <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <version>3.1.1</version>
```

Comment:
the version of the plugin should be specified in the main pom unless we have a good reason to override the version only in this extension

Reply:
The version is the same as in the main POM, I will remove the version as it is redundant here.